examples/with-rollup/README.md

```diff
@@ -0,0 +1,72 @@
+# Turborepo starter with Rollup
```

Comment:
This should include something about how to use this folder, as people are expecting to come to examples and use it as a template. Similar to the design-system example: https://github.com/vercel/turborepo/blob/main/examples/design-system/README.md#getting-started.